system-variables.md

```diff
@@ -1332,6 +1332,10 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
 >
 > Before you upgrade TiDB to v6.5.0 or later, it is recommended that you check whether the [`temp-dir`](/tidb-configuration-file.md#temp-dir-new-in-v630) path of TiDB is correctly mounted to an SSD disk. This path is a TiDB configuration item, which takes effect after TiDB is restarted. Therefore, setting this configuration item before upgrading can avoid another restart.
 
+> **Note:**
+>
+> Index acceleration requires a [`temp-dir`](/tidb-configuration-file.md#temp-dir-new-in-v630) that is writable and has enough free space. If the `temp-dir` is unusable, TiDB falls back to non-accelerated index building. It is recommended to put the `temp-dir` on a SSD disk.
```

Comment:
Can you please merge the two notes into one? Like using a list:

> Note:
> - one note...
> - another note...

The rest LGTM

Reply:
Done.